resolwe/flow/managers/listener.py

```diff
@@ -73,9 +85,103 @@
 logger = logging.getLogger(__name__)
 
 
-class PythonProcessMixin:
+class ListenerPlugin(metaclass=abc.ABCMeta):
+    """Base class for listener plugin classes."""
+
+    @classmethod
+    def __init_subclass__(cls: Type["ListenerPlugin"], **kwargs):
+        """Register class with the registry on initialization."""
+        super().__init_subclass__(**kwargs)
```

Comment:
Do you need the `super` call here?

Reply:
Not for my own use. However, there may be some Python machinery depending on it.